repo/packages/B/beakerx/3/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.9",
+  "name": "beakerx",
+  "version": "0.16.1",
```

Comment:
I would make the version in the format `baseVersion`-`packageVersion`, i.e., 0.16.1-1